stingray/crossspectrum.py

```diff
@@ -565,9 +565,10 @@ def __init__(self, lc1=None, lc2=None, segment_size=None,
         """
         self.type = "crossspectrum"
 
-        if segment_size is not None:
```

Comment:
This is because we need to be able to create empty cross spectra.
Now it should be more clear. If you specify the light curves, you also need to specify the segment size, or it will fail. If you instantiate an empty averaged spectrum, then it is allowed.